apps/web/src/pages/Profile/AccountSettings/RecruitmentAvailability.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { PoolCandidateStatus, User } from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Link, Well } from "@gc-digital-talent/ui";
+
+import ApplicationCard from "~/pages/ProfileAndApplicationsPage/components/TrackApplications/ApplicationCard";
+import useRoutes from "~/hooks/useRoutes";
+
+const inlineLink = (href: string, chunks: React.ReactNode) => (
+  <Link href={href} color="black">
+    {chunks}
+  </Link>
+);
+
+const RecruitmentAvailability = ({ user }: { user: User }) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const activeApplications = unpackMaybes(user.poolCandidates).filter(
+    ({ status }) =>
+      status &&
+      [
+        PoolCandidateStatus.QualifiedAvailable,
+        PoolCandidateStatus.QualifiedUnavailable,
+        PoolCandidateStatus.QualifiedWithdrew,
+        PoolCandidateStatus.PlacedTentative,
+        PoolCandidateStatus.PlacedCasual,
+        PoolCandidateStatus.PlacedIndeterminate,
+        PoolCandidateStatus.PlacedTerm,
+        PoolCandidateStatus.Expired,
+      ].includes(status),
+  );
+  return activeApplications.length > 0 ? (
+    activeApplications.map((application) => (
+      <ApplicationCard key={application.id} application={application} />
+    ))
+  ) : (
+    <Well data-h2-margin="base(x1 0)" data-h2-text-align="base(center)">
+      <p data-h2-margin="base(0, 0, x1, 0)" data-h2-font-weight="base(bold)">
+        {intl.formatMessage({
+          defaultMessage:
+            "There aren't any recruitment processes on your profile yet.",
+          id: "xCJW2N",
+          description:
+            "Message displayed in recruitment availability when the user is not in any valid pools",
+        })}
```

Comment:
This sentence doesn't seem accurate. I can have recruitment processes but they just aren't active. Should we add the word "active" here?

"There aren't any _active_ recruitment processes..."

Reply:
@substrae Any comments on this?

Reply:
Don't inactive ones still display as expired/archived?

Reply:
Okay 👍 Fixed by [0982b05](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/0982b05a0bca56a803d8fc74f7a5d701904bcea8)